packages/vuetify/src/components/VSlideGroup/VSlideGroup.ts

```diff
@@ -315,12 +322,13 @@ export const BaseSlideGroup = mixins<options &
       }
     },
     scrollTo /* istanbul ignore next */ (location: 'prev' | 'next') {
-      this.scrollOffset = this.newOffset(location)
-    },
-    setOverflow () {
-      this.isOverflowing = this.widths.wrapper < this.widths.content
+      this.scrollOffset = this.calculateNewOffset(location, {
+        // Force reflow
+        content: this.$refs.content ? this.$refs.content.clientWidth : 0,
+        wrapper: this.$refs.wrapper ? this.$refs.wrapper.clientWidth : 0
+      }, this.$vuetify.rtl, this.scrollOffset)
     },
-    setWidths () {
+    setWidths /* istanbul ignore next */  () {
```

Comment:
Why ignoring?

Reply:
Same reason as why `scrollTo` and `newOffset` were previously ignored. Not sure what was the actual reason though :P